Terminal Commands.md

```diff
@@ -22,8 +28,18 @@ Some commands can be found in the `bin` directory.
 `mkdir` - makes a directory<br/>
 `rmdir` - removes an **empty** directory<br/>
 
+`cp file1 file2` - used to copy file1 to file2.<br>
+`mv file1 file2` - used to move file1 to file2.
+
+- **Trivia:** `mv oldfilename newfilename` is the best method to rename a file.
+
 `grep <text to search> <files to search in>`
 
+`cut` is used to cut a part of a file<br>
```

Comment:
Ok, we can keep this but update the description to
`cuts out selected portions of each line from file and writes them to the standard output.`

Cut a part of file sounds awkward :P